### PR TITLE
Reinstate Liquidity Rewards Program

### PR DESCRIPTION
**Pull request by hhio618**
This will revert the changes made in commit https://github.com/nation3/citizen-app/commit/0d32b32d14abfb641907c30a2828cdd30c3794ef#diff-c929e9424c96a55187bf3c7ecec95438b67f7a6f20df177e368728ba4bbe4890

## Related GitHub Issue

#225

## Screenshots (if appropriate):

liquidity

## How Has This Been Tested?

- [X] Status checks pass (lint, build, test)
- [X] Works on Sepolia preview deployment
- [x] Works on Mainnet preview deployment

### CODE REVIEW

**Review by aahna-ashina — CHANGES_REQUESTED** [review]
@hhio618 I tested on the Mainnet preview:  https://citizen-app-mainnet-git-fork-hhio618-main-nation3dao.vercel.app

On the `/lock` page, my $veNATION balance shows as 19.65:

> image

But on the `/liquidity` page it shows as 0:

> liquidity

Any idea why the $veNATION balance is not loading?

**Comment by hhio618** [comment]
Please test using the latest commit.

File: ui/pages/liquidity.tsx [hunk 1]
```diff
@@ -1,62 +1,222 @@
...
-        <p>Our liquidity rewards program is over.</p>
         <p>
-          Please{' '}
-          <span className="font-semibold">
-            withdraw your LP tokens and rewards
-          </span>{' '}
-          as soon as possible.
+          Provide liquidity in the pool and then deposit the pool token here.{' '}
+          <GradientLink
+            href={`${balancerDomain}/#/pool/${balancerPoolId}`}
```
Comment (aahna-ashina) [inline ui/pages/liquidity.tsx:102]:
```suggestion
            href={`${balancerDomain}/pool/${balancerPoolId}`}
```

**Review by aahna-ashina — CHANGES_REQUESTED** [review]

**Comment by aahna-ashina** [comment]
@hhio618 I'm getting a `division-by-zero` error at https://citizen-app-sepolia-i57i145rj-nation3dao.vercel.app/liquidity

Is that because of the Reward APY being zero?

>

**Comment by hhio618** [comment]
@aahna-ashina I'm not sure! Please send me the testnet token so that I can test this further.
My eth address: 0x6321286F9B73f427C72e1f9F1bC6b3d25eF06605

**Comment by aahna-ashina** [comment]
> Please send me the testnet token so that I can test this further.

@hhio618 Here is how to get testnet tokens:  https://github.com/nation3/.github/blob/main/CONTRIBUTING.md

**Comment by hhio618** [comment]
@aahna-ashina Found the issue, the totalDeposit is zero in this contract: https://sepolia.etherscan.io/address/0x5514cF24D2241Ecc6012306927eA8d74E052416D#readContract. This value is being used for division in this code: https://github.com/nation3/citizen-app/blob/833fc2215aea6fa42afda8d76c6b9d63b1faee21/ui/lib/liquidity-rewards.ts#L75.

**Comment by aahna-ashina** [comment]
> Found the issue, the totalDeposit is zero in this contract: https://sepolia.etherscan.io/address/0x5514cF24D2241Ecc6012306927eA8d74E052416D#readContract. This value is being used for division in this code: https://github.com/nation3/citizen-app/blob/833fc2215aea6fa42afda8d76c6b9d63b1faee21/ui/lib/liquidity-rewards.ts#L75.

@hhio618 Okay, thank you.  So if we make a deposit, the error message should disappear?

**Comment by hhio618** [comment]
@aahna-ashina You're welcome! Yes, Just add tokens using the `deposit` method, which will do the trick!

**Review by aahna-ashina — APPROVED** [review]

**Pull request merged**